data/sites.json

```diff
@@ -325,5 +325,13 @@
                 "zip": "01915"
             }
         ]
+    },
+    "SouthLawrenceEastElementarySchool": {
+        "name": "South Lawrence East Elementary School",
+        "website": "https://lawrencegeneralcovidvaccine.as.me/schedule.php",
+        "address": "165 Crawford Street",
+        "city": "Lawrence",
+        "state": "MA",
+        "zip": " 01843"
```

Comment:
- Nitpick: leading zero on zip code. 
- Also please move this block into a `SouthLawrence/config.js`

Reply:
- hopefully it was the leading space not 0 that was the problem :)
- moved